lib/galaxy/webapps/tool_shed/config.py

```diff
@@ -66,7 +66,7 @@ def __init__( self, **kwargs ):
         self.new_file_path = resolve_path( kwargs.get( "new_file_path", "database/tmp" ), self.root )
         self.cookie_path = kwargs.get( "cookie_path", "/" )
         self.enable_quotas = string_as_bool( kwargs.get( 'enable_quotas', False ) )
-        self.id_secret = kwargs.get( "id_secret", "USING THE DEFAULT IS NOT SECURE!" )
+        self.id_secret = kwargs.get( "id_secret", "changethisinproductiontoo")
```

Comment:
It can - but I have commented out the option in tool shed ini as part of this commit. So either we change default for people that copied the sample over and didn't touch it or we change the default for people for people who commented out the sample value. I'd guess the previous one is much, much more common. We made a switch like this when we commented out the value in galaxy.ini.sample and I'm pretty sure no one ever complained.